src/main/java/com/jprotractor/NgWebDriver.java

```diff
@@ -11,137 +10,125 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.internal.WrapsDriver;
 
-public class NgWebDriver implements WebDriver, WrapsDriver {
-    private WebDriver driver;
-    private JavascriptExecutor jsExecutor;
-    private String rootElement;
-    @SuppressWarnings("unused")
-    private NgModule[] mockModules;
-    public boolean ignoresync;
-    // little max iteration count for protector
-    public int maxIterations;
-    private int iterationCount;
-
-    public NgWebDriver(WebDriver driver) {
-        this.driver = driver;
-        this.jsExecutor = (JavascriptExecutor) driver;
-        this.rootElement = "body";
-        maxIterations = 30;
-        iterationCount = 0;
-    }
-
-    public NgWebDriver(final WebDriver driver, final boolean ignoreSync) {
-        this(driver);
-        this.ignoresync = ignoreSync;
-    }
-
-    public NgWebDriver(final WebDriver driver, final NgModule[] mockModules) {
-        this(driver, "body", mockModules);
-    }
-
+/**
+ * Angular WebDriver Implementation.
+ */
+public final class NgWebDriver implements WebDriver, WrapsDriver {
+    private final WebDriver driver;
+    private final JavascriptExecutor jsExecutor;
+    private final String root;
+
+    /**
+     * Ctor.
+     * @param drv Parent web driver.
+     */
+    public NgWebDriver(final WebDriver drv) {
+        this(drv, "body");
```

Comment:
we can change all of this later, I'm trying to do the less possible just to make qulice pass, and them we can improve each piece of code...